content/posts/jenkins-plugins-good-bad-ugly.md

```diff
@@ -10,7 +10,7 @@ exif: "SONY RX-100 ISO 125 32.17mm ƒ/5.6 1/400"
 draft: false
 tags: ["jenkins","plugins","containers","CasC"]
 ---
-There are over 1400 Jenkins plugins and that is both a blessing and a curse. Of those 1400 plugins only a small percentage are well maintained and tested, and even fewer (140 of 1400+) are part of the [CloudBees Assurance Program (CAP)](https://go.cloudbees.com/docs/cloudbees-documentation/assurance-program/) as verified and/or compatible plugins - well tested to interoperate with the rest of the CAP plugins (and their dependencies) and with a specific LTS version of Jenkins. Problems can arise when you use plugins that aren't part of CAP, or a plugin that isn't well maintained or tested to work with all of the other plugins you are using and the specific version of Jenkins that you are using. But the extensibility offered by plugins has helped make Jenkins the most popular CI tool on the planet.
```

Comment:
Side note: Jenkins is not branded on the Jenkins Project as a CI since many years, but an "automation tool". At least please use CI/CD (this comment is more for @kmadel than for Oleg)

Reply:
Sure, but most of its popularity comes from the idea that it is a CI tool - at least that is my opinion. This post isn't documentation, but an opinion.

Reply:
IMO, reducing Jenkins to its CI capabilities is downplaying it. Also:

https://github.com/jenkinsci/jenkins#about

![image](https://user-images.githubusercontent.com/223853/59032202-fa10f300-8865-11e9-8bf1-d669e54bfbe8.png)